packages/shared/locales/en.json

```
@@ -1138,8 +1138,8 @@
         "sendPayment": "Send Payment",
         "moveFunds": "Internal Transfer",
         "sendTo": "Send To",
-        "sendFunds": "Send funds",
-        "sendTokensToAddress": "Send tokens to an address",
+        "sendAssets": "Send assets",
+        "sendAssetToAddress": "Send asset to an address or wallet",
```

Technically address covers wallet. Not sure if it's worth stretching this sentence, since it might look ugly on other languages.

---

Fair enough looked a bit short in english, I meant an internal wallet because you can use both in the recipient selector:
```suggestion
        "sendAssetToAddress": "Send asset to an address",
```